modules/era/sql_dynamis/era_dyna_sql.sql

```diff
@@ -458,38 +458,38 @@ REPLACE INTO `mob_droplist` VALUES (265,0,0,1000,1449,100); -- Whiteshell
 REPLACE INTO `mob_droplist` VALUES (265,2,0,1000,1449,150); -- Whiteshell (Steal)
 REPLACE INTO `mob_droplist` VALUES (265,0,0,1000,1450,50); -- Jadeshell
 DELETE FROM `mob_droplist` WHERE dropid = "559"; -- Dagourmarche
-REPLACE INTO `mob_droplist` VALUES (559,0,0,1000,1560,300); -- Attestation of Bravery
-REPLACE INTO `mob_droplist` VALUES (559,0,0,1000,1563,300); -- Attestation of Fortitude
-REPLACE INTO `mob_droplist` VALUES (559,0,0,1000,1567,300); -- Attestation of Virtue
+REPLACE INTO `mob_droplist` VALUES (559,1,1,300,1560,100); -- Attestation of Bravery
+REPLACE INTO `mob_droplist` VALUES (559,1,1,300,1563,100); -- Attestation of Fortitude
+REPLACE INTO `mob_droplist` VALUES (559,1,1,300,1567,100); -- Attestation of Virtue
```

Comment:
Shouldn't these have a group drop rate of 100% and 333/334 for the attestations

As is, isn't this a 30% chance for the group then 10% chance at each attestation?